src/cuga/cli.py

```
@@ -162,7 +166,7 @@ def wait_for_server(
                 )
 
 
-def wait_for_registry_server(port: int, max_retries: int = 120, retry_interval: float = 0.5):
+def wait_for_registry_server(port: int, max_retries: int = None, retry_interval: float = 0.5):
```

Ruff (0.14.8)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)